Modelica/Electrical/Machines.mo

```diff
@@ -4713,6 +4713,7 @@ So the machine is at the beginning in cold condition, ending in warm condition
       end DCPM_Temperature;
 
       model DCPM_Cooling "Test example: Cooling of a DCPM motor"
+        import Modelica;
```

Comment:
We want to get rid of useless imports.

Reply:
Done by my last commit/push to my fork/branch